### PR TITLE
Unity 2019.2.0f1 upgrade

### PR DESCRIPTION
**Pull request by zeroZshadow**
This update removes the Android preparation step, which is no longer required since 2019.2
It also fixes the script that scrapes the required unity version from the project.

### CODE REVIEW

File: scripts/build-worker.sh [hunk 1]
```diff
@@ -18,12 +18,6 @@ fi
 source "$(dirname "$0")/pinned-tools.sh"
 
 pushd "$(dirname "$0")/../"
-    if [[ "${WORKER_TYPE}" == "MobileClient" ]]; then
```
Comment (jamiebrynes7) [inline scripts/build-worker.sh:21]: 🎉

**Review by jamiebrynes7 — APPROVED** [review]
Looks good. Lets get the blank and FPS upgrade PRs in place before we merge this :)

File: tools/UnityPaths/UnityPaths.cs [hunk 2]
```diff
@@ -65,7 +83,7 @@ public static string TryGetUnityPath()
             }
 
             throw new Exception(
-                $"Could not find Unity in\n  {UnityHomeEnvVar}={UnityHomeValue}\n  {improbableUnityPath}\n  {hubPath}\n  {DefaultUnityPath}");
+                $"Could not find Unity in\n  [{improbableUnityPath}]\n  [{hubPath}]\n  [{DefaultUnityPath}]");
```
Comment (jessicafalk) [inline tools/UnityPaths/UnityPaths.cs]:
```suggestion
                $"Could not find Unity in\n[{improbableUnityPath}]\n[{hubPath}]\n[{DefaultUnityPath}]");
```

nit: spaces are not really needed?

**Review by jessicafalk — APPROVED** [review]

File: tools/UnityPaths/UnityPaths.cs (hunk 2 above)
Comment (zeroZshadow) [reply tools/UnityPaths/UnityPaths.cs]: It just indents the paths in the log files, rather unneeded.

Comment (zeroZshadow) [reply tools/UnityPaths/UnityPaths.cs]: Will do

**Pull request merged**